src/java/org/apache/cassandra/tools/nodetool/GcStats.java

```diff
@@ -31,7 +39,69 @@ public void execute(NodeProbe probe)
         double[] stats = probe.getAndResetGCStats();
         double mean = stats[2] / stats[5];
         double stdev = Math.sqrt((stats[3] / stats[5]) - (mean * mean));
+        long[] thread = probe.numberOfGCThreads();
+
+        //value of jnaLockable
+        boolean jnaLockable = NativeLibrary.jnaMemoryLockable();
+
+        OperatingSystemMXBean osBean = (OperatingSystemMXBean) ManagementFactory.getOperatingSystemMXBean();
+
+        long freeMemoryBytes = osBean.getFreeMemorySize();
+        long swapMemoryBytes = osBean.getFreeSwapSpaceSize();
+        long totalMemoryBytes = osBean.getTotalMemorySize();
+        long totalswapMemoryBytes = osBean.getTotalSwapSpaceSize();
+
+        String PM_freeMemoryBytes = FBUtilities.prettyPrintMemory(freeMemoryBytes, " ");
+        String PM_totalMemoryBytes = FBUtilities.prettyPrintMemory(totalMemoryBytes, " ");
+        String PM_totalswapMemoryBytes = FBUtilities.prettyPrintMemory(totalswapMemoryBytes, " ");
+
+        String PM_OSInUseMemoryBytes = FBUtilities.prettyPrintMemory(totalswapMemoryBytes - freeMemoryBytes, " ");
+        String PM_SWAPInUseMemoryBytes = FBUtilities.prettyPrintMemory(totalswapMemoryBytes - swapMemoryBytes, " ");
+
+        probe.output().out.println("GC Threads: " + thread[1]);
+        probe.output().out.println("Duration: " + thread[0] + " ms");
+        probe.output().out.println("MemLock: " + jnaLockable + "\n");
+
+        long osInUse = totalMemoryBytes - freeMemoryBytes;
```

Comment:
Shouldn't we rename `osInUse` to something like `osMemoryInUse?`

Reply:
done!